packages/typespec-test/test/loadTest/spec/models.tsp

```diff
@@ -501,7 +501,6 @@ model ResourceMetric {
 model TestRuns {
   // FIXME: is testRunId optional or required in the response? Resource key and path params cannot be optional
   @key
-  @path
```

Comment:
This will cause testRunId being filtered in the RLC response type. which will cause compile errors and if we need to fix it, it will require us to support visibility which is a lot of work. so I removed @ path  temporarily, and also confirmed with Catalina offline, she will rewrite this part.